packages/zoe/src/state.js

```diff
@@ -27,12 +27,12 @@ const makeInstanceTable = () => {
 };
 
 // Offer Table
-// Columns: handle | instanceHandle | proposal | currentAllocation
+// Columns: handle | instanceHandle | proposal | currentAllocation, notifier
 const makeOfferTable = () => {
   // TODO: make sure this validate function protects against malicious
   // misshapen objects rather than just a general check.
   const validateProperties = makeValidateProperties(
-    harden(['instanceHandle', 'proposal', 'currentAllocation']),
+    harden(['instanceHandle', 'proposal', 'currentAllocation', 'notifier']),
```

Comment:
It seems like we should divide the notifier into two facets: an admin facet held only by Zoe and a `getUpdateSince` read-only facet. By putting the entire notifier here, we're making the admin facet available to everyone who can get the offerRecord - which means anyone could update the notifier state. That doesn't seem right. 

I think we need to filter out the admin facet part of the notifier from anything that gets to the user in the Zoe Service or the contract code in the ZCF. There's a helper function to do so (filterObj) - `removePurse` uses it.

Reply:
Yes, you're right.